packages/transport/src/core/CoreErrors.ts

```diff
@@ -197,6 +197,13 @@ class General {
     public invalidTruncatedReference() {
         return new CoreError("error.transport.files.invalidTruncatedReference", "invalid truncated reference");
     }
+
+    public runtimeVersionIncompatibleWithBackboneVersion(backboneVersion: number) {
+        return new CoreError(
+            "error.transport.files.runtimeVersionIncompatibleWithBackboneVersion",
```

Comment:
Should be `.general.` instead of `.files.`. Some other error codes here are inconsistent as well, but this is unrelated to the PR. Can we change it in a separate PR or would this be a breaking change such that we leave it for now?